docker/transport/unixconn.py

```diff
@@ -5,16 +5,14 @@
 from docker.transport.basehttpadapter import BaseHTTPAdapter
 from .. import constants
 
-try:
-    import requests.packages.urllib3 as urllib3
-except ImportError:
-    import urllib3
+import urllib3
+import urllib3.connection
 
 
 RecentlyUsedContainer = urllib3._collections.RecentlyUsedContainer
 
 
-class UnixHTTPConnection(httplib.HTTPConnection):
+class UnixHTTPConnection(urllib3.connection.HTTPConnection):
```

Comment:
Thanks! I can't comment on unedited lines, but wanted to mention that putheader() and response_class() can both be removed.

Reply:
Thanks! I've removed these functions in a48dd4c0cf0c34e3d15c2676d23349842af0ad53.